src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -18,10 +18,20 @@ import {CalibrateResultWithType, Dict, ModelOutputTabs} from "../../types";
 import {DownloadResultsMutation} from "../downloadResults/mutations";
 import {PlottingSelectionsMutations} from "../plottingSelections/mutations";
 import { ModelOutputMutation } from "../modelOutput/mutations";
+import { BarchartSelections, BubblePlotSelections, ChoroplethSelections, TableSelections } from "../plottingSelections/plottingSelections";
 
-type ResultDataPayload = {
-    indicatorId: string,
-    tab: ModelOutputTabs
+export type ResultDataPayload = {
+    payload: Partial<BarchartSelections>,
```

Comment:
`payload.payload` feels a bit weird, how about just renaming to `selections` for these types?
```suggestion
    selections: Partial<BarchartSelections>,
```